src/table.js

```diff
@@ -672,7 +678,8 @@ export const createTable = (
   schema,
   rowsCount = 3,
   colsCount = 3,
-  withHeaderRow = true
+  withHeaderRow = true,
+  withDefaultNodeContent = null
```

Comment:
I know that naming is hard, but by looking at `withDefaultNodeContent`  I'd expect it to be of type boolean. Lets change that to `cellContent`